Application/EdFi.Ods.AdminApi.V1/Infrastructure/Database/EntityFrameworkCoreDatabaseModelBuilderExtensions.cs

```diff
@@ -30,7 +30,8 @@ public static void ApplyDatabaseServerSpecificConventions(this ModelBuilder mode
             {
                 var tableId = StoreObjectIdentifier.Table(tableName);
                 var columnName = property.GetColumnName(tableId) ?? property.GetDefaultColumnName(tableId);
-                property.SetColumnName(columnName.ToLowerInvariant());
+                if (columnName != null)
+                    property.SetColumnName(columnName.ToLowerInvariant());
```

Comment:
I'm surprised this was accepted without `{ }`. I thought our rules did not allow that.